vyos/vyos-1x
# monitoring: T4312: Ability to set IP address in the URL

## Change Summary

 Ability to set IP address in URL

## Types of changes


## Related Task(s)

* https://phabricator.vyos.net/T4312

## Component(s) name

monitoring, influxdb
## Proposed changes


## How to test
```
set service monitoring telegraf influxdb authentication organization 'log@in.local'
set service monitoring telegraf influxdb authentication token 'GuRJc12tIzfjnYdKRAIYbxdWd2aTpOT9PVYNddzDnFV4HkAcD7u7-kndTFXjGuXzJN6TTxmrvPODB4mnFcseDV=='
set service monitoring telegraf influxdb port '8086'
set service monitoring telegraf influxdb url 'http://192.0.2.54'
```
Expected config:
```
vyos@r14# cat /run/telegraf/telegraf.conf | grep url -B 1
[[outputs.influxdb_v2]]
  urls = ["http://192.0.2.54:8086"]
[edit]
vyos@r14# 
```

## Checklist:
